shared/tab-bar/index.render.native.js

```diff
@@ -0,0 +1,47 @@
+// @flow
```

Comment:
This file just provides a simpler interface to create the main tab bar of the app.